README.md

```diff
@@ -1 +1,31 @@
-# animal-shelter
\ No newline at end of file
+# Animal shelter
+
+## Motivation
+
+The idea of this project is to help with understanding and consolidate knowledge about different technologies used in programming. 
+
+## About project
+
+### Aim
+The project was created to learn and practice using these skills in a real platform with real problems. 
+
+The `Animal shelter` is an application that focuses on simple operations in the beginning and in process of time will be developed and new functionalities will be added. 
+
+### The main goal
+The `Animal shelter` stores data about animals and employees (zookeepers, doctors etc). Both animals and employees can be added, delete, update, and read data. 
+
+### Technologies
+In this project the following technologies will be used:
+*Java 11, Spring, H2 database (in the beginning), Mockito, Junit.*
+
+The technologies were chosen because of their wide use in the market.
+
+## How to run the project
+*TBD*
+
+## How to run tests
+*TBD*
+
+
```

Comment:
Maybe we can add authors paragraph here.

Reply:
Do you think is it necessary?

Reply:
It was just a proposition. We can stay without it.